Gemfile

```diff
@@ -30,7 +30,7 @@ group(:development, :test) do
   # gem 'chefstyle', github: 'chef/chefstyle'
   gem "chefstyle", git: "https://github.com/chef/chefstyle.git", branch: "master"
 
-  gem "ruby-shadow", platforms: :ruby unless RUBY_PLATFORM.downcase.match(/(aix|cygwin)/)
+  gem "ruby-shadow", platforms: :ruby unless RUBY_PLATFORM.downcase =~ /(aix|cygwin)/
```

Comment:
Not a lot:  https://github.com/JuanitoFatas/fast-ruby#regexp-vs-stringmatch-vs-string-code-

I kind of like =~ better than match -- i've probably been infected by writing too many lines of Perl in my life though....